TestCases/optimization_euler/equivalentarea_naca64206/NACA64206.cfg

```diff
@@ -0,0 +1,144 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: NACA64-206 in inviscid, supersonic flow                  %
+% Author: Yuki Utsumi	      	      	              	       	       %
+% Institution: Individual				               %
+% Date: 2021.08.07                                                            %
+% File Version 7.1.1 "Blackbird"                                                %
+%                                                                              %
```

Comment:
Do you want to make this all pretty? :)

Reply:
Not really. I just tried to mimic some of existing config files in TestCases. I have deleted those lines.

Reply:
Hmm ok I was just pointing out that the "%" were not aligned, which can happen if you used tabs.

Reply:
Ah, I misunderstood. I brought back those lines and aligned "%".